### PR TITLE
Pyt 1663 fastapi

### PR DESCRIPTION
**Pull request by danilito19**
Helpful app for testing and future implementation.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,3 +33,10 @@ wsgi-two-apps: templates
 
 bottle: templates
 	python apps/bottle_app.py $(HOST) $(PORT)
+
+# why does this not work?
```
Comment (danilito19) [inline Makefile:37]: i really don't know why it doesn't work to turn on the server, but I'm fine leaving it as is and when we do actually add fastapi support to figure it out

File: setup.py [hunk 2]
```diff
@@ -17,6 +17,7 @@
     "falcon-multipart==0.2.0",
 } | trigger_extras
 flask_extras = {"Flask<3"} | trigger_extras
+fastapi_extrast = {"fastapi==0.68.0", "uvicorn[standard]"} | trigger_extras
```
Comment (lazorchakp) [inline setup.py:20]: typo `fastapi_extrast`

**Review by lazorchakp — APPROVED** [review]
I think we've officially backed ourselves into a corner with the apps here. Logically I don't think it makes sense for this app to go in this repo - it doesn't even import `vulnpy`. It all looks great, and I'm sure it will be very useful going forward, but I think we're starting to abuse this project a bit.

So I'm ok with this going through, but I'm going to think about our integration testing architecture a little and hopefully come up with something that keeps vulnpy a little more clean

**Comment by danilito19** [comment]
> I think we've officially backed ourselves into a corner with the apps here. Logically I don't think it makes sense for this app to go in this repo - it doesn't even import `vulnpy`. It all looks great, and I'm sure it will be very useful going forward, but I think we're starting to abuse this project a bit.
> 
> So I'm ok with this going through, but I'm going to think about our integration testing architecture a little and hopefully come up with something that keeps vulnpy a little more clean

I know what you mean with we've started to abuse this repo a bit, but I actually think the fastapi app does make sense here. we will be using it for testing. the views I've added are the abuse... but it's the best way I have to document and share with you all how this stuff works.

**Comment by lazorchakp** [comment]
@danilito19 just to be clear, I'm not trying to say you've done something wrong. I think this is a great way to catch everyone up on what you've learned as quickly as possible

**Pull request merged**